python-backend/app/api/users/minespace/resources/minespace_user.py

```diff
@@ -26,7 +27,18 @@ def get(self, user_id=None):
                     return self.create_error_payload(404, "user not found"), 404
             result = user.json()
         else:  #get all users (not deleted)
-            result = {'users': [x.json() for x in MinespaceUser.get_all()]}
+            ms_users = MinespaceUser.get_all()
+            all_mines = []
+            for user in ms_users:
+                for user_mine in user.mines:
+                    all_mines.append(str(user_mine.mine_guid))
+
+            mine_basic_info_resp = requests.post(
```

Comment:
Why not just use the response with all minespace users (the 'before' in this diff) to fetch the basic mine data with an API call to '/basicinfo' from the frontend?

I get the impression here that this is a concern for the frontend (displaying mine names for mines associated with users -- which is what I think this change is meant to do). The backend shouldn't care that the frontend wants to do that.

Correct me if I'm misunderstanding the goal of this PR.

Reply:
That would be the other approach, both are valid, I went this way because the endpoint is not useful without this information also being returned, so I figured it would just make the hop from the backend.

Reply:
I think this falls in line with what @nayr974 and I were discussing the other day on Slack. This adds the logic for a UI concern to the backend. If we don't want business logic in our endpoints then I think we should be consistent with that decision.

A `mine_guid` is totally sufficient for a consumer of the API to use to then fetch whatever mine data they want. Fetching that data for the consumer on the backend is preemptive and should be avoided, in my opinion.

Reply:
Working right now to change this to the frontend fetching this.

Reply:
Ryan said this was addressed IRL. Resolving the conversation

Reply:
@NWCalvank Changed it so frontend makes the second call, same pattern as PartyRelationships. Let me know if it looks good.